### PR TITLE
Initial build: Governance homepage

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: governance/index.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+    <head>
+        <title>Goverance</title>
```
Comment (BlueReZZ) [inline governance/index.html]: Until we have a full site perhaps this should be `Libero Goverance`

File: governance/index.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" type="text/css" media="screen" href="assets/css/all.css" />
+    </head>
+    <body>
+    <div class="global-wrapper">
+      <main class="main">
+  		<div class="wrapper wrapper--content">
+  			<h1 class="content-header__title">Governance</h1>
```
Comment (BlueReZZ) [inline governance/index.html]: Perhaps `Libero Governance` would be better until the whole site is made

File: governance/index.html [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+        <title>Goverance</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" type="text/css" media="screen" href="assets/css/all.css" />
+    </head>
+    <body>
+    <div class="global-wrapper">
+      <main class="main">
+  		<div class="wrapper wrapper--content">
```
Comment (BlueReZZ) [inline governance/index.html]: The default width seems wide for such a large amount of text. Could a similar width the `labs` posts be used instead, for example https://elifesciences.org/labs/0edc1d8c/evaluating-ideas-for-elife-s-innovation-initiative

File: governance/index.html [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+  			<h1 class="content-header__title">Governance</h1>
+
+  			<p class="lead-para" id="iAmTheOptionalId">Libero is eLife’s [1] platform to help publishers do more with everything they publish. The platform has evolved since development began in 2015 from an enabler of innovation for eLife’s own journal, to a more reusable and modern platform for publishing, and now as a community-driven platform for publishing and presentation of scholarly content.</p>
```
Comment (BlueReZZ) [inline governance/index.html]: Can you link the `[1]` to the footnote, please?

**Review by BlueReZZ — CHANGES_REQUESTED** [review]
Overall this looks great. There's a few minor tweaks we should do now, and then we can iterate on the links once the other pages exist

File: governance/index.html (hunk 1 above)
Comment (discodavey) [reply governance/index.html]: Title is now "Libero Governance"

File: governance/index.html (hunk 2 above)
Comment (discodavey) [reply governance/index.html]: H1 tag is now Libero Governance

File: governance/index.html (hunk 3 above)
Comment (discodavey) [reply governance/index.html]: Grid is in page and content is as example.

File: governance/index.html (hunk 4 above)
Comment (discodavey) [reply governance/index.html]: Added anchor so will navigate to footer

**Comment by discodavey** [comment]
I have at the moment added the eLife favicon until we create the new one's using the Libero logo.

**Comment by discodavey** [comment]
Committed changes as discussed above.

File: governance/index.html [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+              <section class="article-section" data-behaviour="ArticleSection">
+
+                <header class="article-section__header">
+                  <h2 class="article-section__header_text">Dispute Resolution</h2>
+                </header>
+
+                <div class="article-section__body">
+                  <p>As suggested by <a href="http://oss-watch.ac.uk/resources/benevolentdictatorgovernancemodel">OSS Watch</a>:</p>
+
+                  <blockquote class="pull-quote">
```
Comment (thewilkybarkid) [inline governance/index.html]: Less fancy, but AFAIK https://ui-patterns.elifesciences.org/?p=molecules-quote is more meant for including content from somewhere else. (As opposed to something someone said.)

File: governance/index.html [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+
+                  <p>The components of the system will be separated into Core components (all supported and the copyright owned by eLife), Extension components (supported and the copyright owned by eLife, where appropriate) and Libero-compatible components (supported and the copyright owned by the creator).</p>
+
+                  <p>The software will be licenced under the permissive MIT license. Contributors will use the CLA to assign copyright of their contributions to core components or appropriate extension components over to eLife and in return retain attribution in the distributed source code. Copyright of documentation will also be assigned in a similar way and made available under a Creative Commons Attribution License (CC_BY 4.0).</p>
+
+                  <p>eLife will also pursue the concepts of trademark licensing for the name `Libero` so that an ecosystem of service providers can grow around the software and the project can gain some level of sustainability from trademark-licensing fees. Anyone is permitted to use the software as the MIT licence permits, and use the documentation as the CC-BY license permits.</p>
```
Comment (thewilkybarkid) [inline governance/index.html]: `` ` `` from Markdown?

File: governance/index.html [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+
+    <div class="global-wrapper">
+      <main class="main">
+
+      <div class="wrapper wrapper--content">
+        <div class="grid">
+          <div class="content-container grid__item one-whole large--ten-twelfths push--large--one-twelfth x-large--eight-twelfths push--x-large--two-twelfths grid-column">
+            <h1 class="content-header__title">Libero Goverance</h1>
+
+            <p class="lead-para" id="iAmTheOptionalId">Libero is eLife’s <a href="#foot-note">[1]</a> platform to help publishers do more with everything they publish. The platform has evolved since development began in 2015 from an enabler of innovation for eLife’s own journal, to a more reusable and modern platform for publishing, and now as a community-driven platform for publishing and presentation of scholarly content.</p>
```
Comment (thewilkybarkid) [inline governance/index.html]: `iAmTheOptionalId` 😆.

Comment (discodavey) [reply governance/index.html]: I took this from the Pattern Sample :-) . I have removed it

File: governance/index.html (hunk 6 above)
Comment (BlueReZZ) [reply governance/index.html]: Yes, probably doesn't need quotes at all

File: governance/index.html [hunk 8]
```diff
@@ -0,0 +1,315 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+    <head>
+        <title>Libero Goverance</title>
```
Comment (thewilkybarkid) [inline governance/index.html]: Governance

File: governance/index.html [hunk 9]
```diff
@@ -0,0 +1,315 @@
...
+    </head>
+    <body>
+
+    <div class="global-wrapper">
+      <main class="main">
+
+      <div class="wrapper wrapper--content">
+        <div class="grid">
+          <div class="content-container grid__item one-whole large--ten-twelfths push--large--one-twelfth x-large--eight-twelfths push--x-large--two-twelfths grid-column">
+            <h1 class="content-header__title">Libero Goverance</h1>
```
Comment (thewilkybarkid) [inline governance/index.html]: Governance!

File: governance/index.html (hunk 5 above)
Comment (BlueReZZ) [reply governance/index.html]: Agreed, that quote is probably more appropriate, we can keep the other one for people singing our praises!

**Review by BlueReZZ — APPROVED** [review]

File: governance/index.html (hunk 5 above)
Comment (discodavey) [reply governance/index.html]: Changed to class="quote"

File: governance/index.html (hunk 6 above)
Comment (discodavey) [reply governance/index.html]: Removed ` as per discussion with Paul

File: governance/index.html (hunk 8 above)
Comment (discodavey) [reply governance/index.html]: Corrected

File: governance/index.html (hunk 9 above)
Comment (discodavey) [reply governance/index.html]: Corrected

**Comment by BlueReZZ** [comment]
Closes https://github.com/libero/libero/issues/44

**Pull request merged**